example/src/App.tsx

```
@@ -1,441 +1,751 @@
+      action: async (logAndUpdateState: (msg: string) => void) => {
+        try {
+          const result = await kit.sendBatches({
+            onlyBatchNames: ['doesnotexistbatch'],
+          });
+          let msg =
+            'SendBatches with nonexistent batch: ' + JSON.stringify(result);
+          if (
+            result &&
+            result.batches &&
+            result.batches['doesnotexistbatch'] &&
+            result.batches['doesnotexistbatch'].isSuccess
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)